pysages/ssages/collective_variables/__init__.py

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020: Pablo Zubieta (see LICENSE.md)
+# Copyright (c) 2021: Ludwig Schneider (see LICENSE.md)
+
+# pylint: disable=unused-import,relative-beyond-top-level
+# flake8: noqa F401
+
+from .angles import Angle, DihedralAngle
+from .shape import RadiusOfGyration, PrincipalMoment, Asphericity, Acylindricity, ShapeAnisotropy
```

Comment:
For API exposure I like to have long list of imports on a tuple spanning multiple lines, so it's easier to track in the commits what is added/removed

Reply:
agreed, and implemented